app/views/settings/language.rb

```diff
@@ -3,5 +3,74 @@ class Language < ApplicationView
     def page_title
       'Europeana Collections - Language Settings'
     end
+
+    def settings
```

Comment:
Assignment Branch Condition size for settings is too high. [23/15]<br>Method has too many lines. [65/10]